Source/CommandManager.cs

```diff
@@ -107,39 +50,90 @@ protected override void OnAwake()
-                Description = functionDescription,
-                Parameters = parameters,
-                MethodTarget = instance,
-                Method = method
-            });
+            // Find commands
+            var commands = Commands.Where(x => x.Name.Contains(str) || x.Description.Contains(str));
+
+            // Build list
+            foreach (var command in commands)
+                if (ColoredFindOutput)
+                {
+                    // Insert background color
+                    sb.Append(RichTextExtensions.ColorInnerString(command.Name, str, "green"));
+                    sb.Append(": ");
+                    sb.Append(RichTextExtensions.ColorInnerString(command.Description, str, "green"));
+
+                    list.Add(sb.ToString());
+                    sb.Clear();
+                }
+                else
+                {
+                    list.Add($"{command.Name}: {command.Description}");
+                }
+
+            // Find config variables
+            var variables = CVarManager.Instance.AllVariables.Where(x =>
+                x.Key.Contains(str) || x.Value.Attribute.Description.Contains(str));
+
+            // Build list
+            foreach (var variable in variables)
+                if (ColoredFindOutput)
+                {
+                    // Insert background color
+                    sb.Append(RichTextExtensions.ColorInnerString(variable.Key, str, "green"));
+                    sb.Append(": ");
+                    sb.Append(RichTextExtensions.ColorInnerString(variable.Value.Attribute.Description, str,
+                        "green"));
+
+                    list.Add(sb.ToString());
+                    sb.Clear();
+                }
+                else
+                {
+                    list.Add($"{variable.Key}: {variable.Value.Attribute.Description}");
+                }
+
+            // Sort list
+            list.Sort();
+            list.ForEach(Debug.Log);
         }
 
-        /// <summary>
-        ///     Gets all commands.
-        /// </summary>
-        /// <returns>The command list.</returns>
-        public static IReadOnlyList<Command> GetCommands()
+        private static object ParseObject(string value, TypeCode type)
         {
-            return Instance._commands;
+            value.ToLower();
+            if (type == TypeCode.DBNull || type == TypeCode.Empty)
+                return null;
+            if (type == TypeCode.String || type == TypeCode.Object)
+                return value;
+            try
+            {
+                if (type == TypeCode.Boolean)
+                {
+                    if (value == "yes" || value == "1" || value == "on")
+                        return true;
+                    if (value == "no" || value == "0" || value == "off")
+                        return false;
+                }
+                return Convert.ChangeType(value, type);
+            }
+            catch (Exception)
```

Comment:
Instead of the try/catch block we can use this method: https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.typeconverter.canconvertfrom?redirectedfrom=MSDN&view=net-5.0#System_ComponentModel_TypeConverter_CanConvertFrom_System_Type_